pkg/config/security/security.go

```diff
@@ -181,3 +182,11 @@ func validateMapKey(key string) error {
 	}
 	return fmt.Errorf("bad key (%s): should have format a[b]", key)
 }
+
+// validCipherSuites contains a list of all ciphers supported in Gateway.server.tls.cipherSuites
+// TODO: DO-NOT-MERGE: extract full supported cipher suite list
+var validCipherSuites = sets.NewSet("ECDHE-ECDSA-AES128-SHA")
```

Comment:
1. "ALL" magical words. user should not use though.
2. `bssl ciphers -openssl-name ALL | rg -v PSK`
ECDHE-ECDSA-AES128-GCM-SHA256
ECDHE-RSA-AES128-GCM-SHA256
ECDHE-ECDSA-AES256-GCM-SHA384
ECDHE-RSA-AES256-GCM-SHA384
ECDHE-ECDSA-CHACHA20-POLY1305
ECDHE-RSA-CHACHA20-POLY1305
ECDHE-ECDSA-AES128-SHA
ECDHE-RSA-AES128-SHA
ECDHE-ECDSA-AES256-SHA
ECDHE-RSA-AES256-SHA
AES128-GCM-SHA256
AES256-GCM-SHA384
AES128-SHA
AES256-SHA
DES-CBC3-SHA

Reply:
Done ,thanks!